packages/visual-editor/src/utils/imagePlaceholders.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Placeholder images for default image placeholders in page sections.
+ */
+const PLACEHOLDER_IMAGE_IDS = [
+  "tiBUJ0Hiwx8", // Crashing ocean wave
+  "OYFHT4X5isg", // Aerial view of pine trees in mist
+  "7Y0NshQLohk", // Sand dune
+] as const;
+
+/**
+ * Gets a random placeholder image URL from the CDN.
+ * @returns A randomly selected placeholder image URL
+ */
+export function getRandomPlaceholderImage(): string {
+  const randomIndex = Math.floor(Math.random() * PLACEHOLDER_IMAGE_IDS.length);
+  const photoId = PLACEHOLDER_IMAGE_IDS[randomIndex];
+  // Using Unsplash's download endpoint which redirects to the CDN
+  // The "Invalid host" error is likely a validation warning but doesn't affect rendering
+  return `https://unsplash.com/photos/${photoId}/download?force=true`;
```

Comment:
Is this the right url? When I put it into my browser it prompts to download the file